src/components/Login/Login.tsx

```diff
@@ -0,0 +1,117 @@
+import { useState } from 'react';
+import Button from 'common/Button/Button';
+import { Input } from 'common/Input/Input';
+import { Link, useNavigate } from 'react-router-dom';
+import { ErrorsParam } from 'helpers/Types';
+import './Login.css';
+
+export const Login = () => {
+  const initialValues = { email: '', password: '', name: '' };
+  const [user, setUser] = useState(initialValues);
+  const [formErrors, setFormErrors] = useState({ email: '', password: '' });
+  const [loginError, setLoginError] = useState('');
+
+  const navigate = useNavigate();
+
+  async function login() {
+    try {
+      const res = await fetch('http://localhost:4000/login', {
```

Comment:
Task-2 still uses mock data, but I'll accept connecting to the backend here since it'll be relevant for the next task

(https://d17btkcdsmqrmh.cloudfront.net/react-fundamentals/docs/module-2/home-task/components#courses)